File: app/src/main/java/com/example/quiz/ScreenActivity.java
package com.example.quiz;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.ImageView;

public class ScreenActivity extends AppCompatActivity {
    ImageView playGame;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_screen);
        //the below method will initialize views
        initViews();

        //PlayGame button - it will take you to the CategoryActivity
        playGame.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent intent = new Intent(ScreenActivity.this, CategoryActivity.class);
                startActivity(intent);
                finish();
            }
        });
    }

    private void initViews() {
        //initialize views here
        playGame =(ImageView)findViewById(R.id.playGame);

    }
}

File: app/src/main/java/com/example/quiz/ScreenActivity.kt
package com.example.quiz

import android.content.Intent
import android.os.Bundle
import android.widget.ImageView
import androidx.appcompat.app.AppCompatActivity
import com.example.quiz.databinding.ActivityScreenBinding
import com.example.quiz.databinding.ActivitySplashScreenBinding

class ScreenActivity : AppCompatActivity() {
    private lateinit var binding: ActivityScreenBinding
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityScreenBinding.inflate(layoutInflater)
        val view = binding.root
        setContentView(view)

        //PlayGame button - it will take you to the CategoryActivity
        binding.playGame.setOnClickListener {
            val intent = Intent(this@ScreenActivity, CategoryActivity::class.java)
            startActivity(intent)
            finish()
        }
    }
}